src/main/java/org/jabref/gui/util/ViewModelListCellFactory.java

```diff
@@ -192,7 +192,10 @@ protected void updateItem(T item, boolean empty) {
                         setOnDragDetected(event -> toOnDragDetected.accept(viewModel, event));
                     }
                     if (toOnDragDropped != null) {
-                        setOnDragDropped(event -> toOnDragDropped.accept(viewModel, event));
+                        setOnDragDropped(event -> {
+                            event.consume(); // Prevent cells from acting as drop targets
+                            getParent().fireEvent(event); // The action performed was not look at the parent, the drop box
```

Comment:
Maybe reformulate the comment - and put it before the statement

Similar below.

```suggestion
                            // The parent is the box for dropping; that should be used as target (and not this cell)
                            event.consume();
                            getParent().fireEvent(event);
```